brainscore_vision/data/freemanziemba2013/data_packaging/2020-02-21_movshon_aperture.ipynb

```diff
@@ -103,8 +103,8 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "brainio-contrib/image_movshon_stimuli.zip: 100%|██████████| 33.5M/33.5M [00:00<00:00, 41.2MB/s]\n",
-      "brainio-contrib/image_movshon_FreemanZiemba2013-public.zip: 100%|██████████| 10.1M/10.1M [00:00<00:00, 16.6MB/s]\n"
+      "brainio-brainscore/image_movshon_stimuli.zip: 100%|██████████| 33.5M/33.5M [00:00<00:00, 41.2MB/s]\n",
+      "brainio-brainscore/image_movshon_FreemanZiemba2013-public.zip: 100%|██████████| 10.1M/10.1M [00:00<00:00, 16.6MB/s]\n"
```

Comment:
this is a weird rename -- maybe from replace all? I don't think we should change notebook outputs (happens again several times after which I'm not going to mark)